src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -3287,6 +3287,8 @@ namespace HVACVariableRefrigerantFlow {
                     if (errFlag) {
                         ShowContinueError("...occurs in " + cCurrentModuleObject + " = " + VRFTU(VRFTUNum).Name);
                         ErrorsFound = true;
+                    } else {
+                        VRFTU(VRFTUNum).fanOutletNode = Fans::Fan(VRFTU(VRFTUNum).FanIndex).OutletNodeNum;
```

Comment:
Save fan outlet node for use in TeTc calculations.

Reply:
Seems reasonable enough, if you are going to have to query what the fan is doing, you can check the outlet node.